### ISSUE TITLE
Pass a taiko script as argument to taiko command

### ISSUE THREAD

**Issue by zabil** [getgauge/taiko#10]
Let the `taiko` command take a taiko script as input to run.

For e.g.
```
$ taiko
> openBrowser("google.com")
> write("Gauge testing")
> click("I'm Feeling Lucky")
> .code gaugesearch.js
$ taiko gaugesearch.js // Runs the tests headlessly
```
`.code` and the saved file can omit the import declaration

```
const { openBrowser, ... } = require('taiko');
```

**Comment by kashishm** [comment]
```
$ taiko
> openBrowser("google.com")
> write("Gauge testing")
> click("I'm Feeling Lucky")
> .code gaugesearch.js
$ node gaugesearch.js // Runs the tests headlessly
```
This works already.

**Comment by zabil** [comment]
Yes, but if we run with
```
> taiko gaugesearch.js
```

We can get rid of the long declaration at the top

```
const { openBrowser, ... } = require('taiko')
```

Also output the result of each action while running on this mode

```
$ taiko googlesearch.js
 ✔ Browser and page initialized
...
```
```

**Comment by kashishm** [comment]
But then it won't work with `node`.

**Comment by zabil** [comment]
That is ok

### Scenario 1
Using taiko to quickly run scripts. Invoke the REPL from any folder, record a script and run using `taiko filename` the resulting script should be short and noise free i.e if the test is only clicking and then other imports are not required. The user may record multiple scripts that can run via taiko

### Scenario 2
When using external modules (require), use `npm` and `package.json` with the line  
```
  "scripts": {
    "test": "taiko gaugesearch.js"
  },
```
This should work for a local install of taiko too.

### Scenario 3
To use it only with node, they add the `require` line write the script using documentation. Use the `.code` feature to copy from the REPL. This works for using it a testing framework like gauge which will have it's own runner.

I see **Scenario 1** as the most common use case.

**Comment by zabil** [comment]
> That is ok

Or

`.code` generates a clean file that *only* imports methods that's recorded.

For .e.g.

```
> openBrower()
> goto('google.com')
> write('gauge site')
> click('Google Search')
> closeBrowser()
> .code
> .code
const { openBrowser, goto, write, click, closeBrowser } = require('taiko');

(async () => {
        await openBrowser();
        await goto('google.com');
        await write("gauge site");
        await click("Google Search");
        await closeBrowser();
})();
```

Note: This is implemented via 0e7096ea71eaff9d526f821e4e425de73df7a12d
